roles/generate_kubernetes_config/templates/panda-pulse.yaml.j2

```diff
@@ -0,0 +1,43 @@
+# {{ ansible_managed }}
+
+panda-pulse:
+  fullnameOverride: "panda-pulse"
+
+  image:
+    repository: {{ default_tooling_images.panda_pulse.split(':') | first }}
+    tag: {{ default_tooling_images.panda_pulse.split(':') | last }}
+    pullPolicy: {% if "latest" in (default_tooling_images.panda_pulse.split(':') | last) %}Always{% else %}IfNotPresent{% endif %}
+
+  schedule: "0 7 * * *"
+
+  config:
+    network: "{{ ethereum_network_name }}"
+    discordChannel: "1207978750004690974"
```

Comment:
could this be an ansible variable and set the value in the defaults so we can override it later?

Reply:
Yeah great shout. Done in [301d27a](https://github.com/ethpandaops/ansible-collection-general/pull/302/commits/301d27aa294a726946e0a580383d8a47f2eb9003).